excel/client.bal

```diff
@@ -0,0 +1,356 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+# Microsoft Excel connector client endpoint
+public client class Client {
```

Comment:
Aren't we gonna add display annotation for client like this?
https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/8ad5e06895588ec206de7b60228a154ed2dd0562/calendar/endpoints.bal#L23

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/commit/5faa81d3bd0c91f10272734f80b7d29c9ee53f63